src/tree.ts

```diff
@@ -103,6 +103,12 @@ export class LaunchTargetNode extends BaseNode {
             item.collapsibleState = vscode.TreeItemCollapsibleState.None;
             item.tooltip = localize("launch.target.currently.selected.for.debug.run.in.terminal",
                                     "The launch target currently selected for debug and run in terminal.\n{0}", this._toolTip);
+            // enablement in makefile.outline.setLaunchConfiguration is not
+            // disabling this TreeItem
+            item.command = {
+                command: "makefile.outline.setLaunchConfiguration",
+                title: "%makefile-tools.command.makefile.setLaunchConfiguration.title%"
+            }

```

Comment:
Until I try this myself to verify, you did not encounter any issues for using "title" and "%" in typescript this way, as opposed to how it's done in package.json? Is the text associated with %makefile-tools.command.makefile.setLaunchConfiguration.title% properly taken from package.nls.json?

Reply:
As far as I can tell, yes.